### ISSUE TITLE
Package configuration not being applied

### ISSUE THREAD

**Issue by frankwallis** [systemjs/systemjs#794]
I have some package configuration in config.js for a remote github package, but it is no longer being applied and so the application is failing because it is attempting to fetch a file with the wrong default extension.

The config is [here](https://github.com/frankwallis/plugin-typescript/blob/latest/example/angular2/config.js#L33) and when debugging the error seems to happen when loading the config (in ```.config```). It looks like normalizeSync is trying to remove a '.js' extension which is not there and so the normalized package name gets truncated to ```http://localhost:8080/jspm_packages/github/frankwallis/example-service@mas``` instead of ```http://localhost:8080/jspm_packages/github/frankwallis/example-service@master``` which means that my package config does not get used later on.

**Comment by guybedford** [comment]
Is there any way you can share a full test case for this? If I clone that test folder can I just run that? Any pointers to the instructions would be great.

So if you look at `System.packages` you can see that it is referenced by the wrong name in that object? It would be good to track this down.

**Comment by frankwallis** [comment]
```
git clone https://github.com/frankwallis/plugin-typescript.git
cd plugin-typescript
npm i
cd examples/angular2
jspm i
```

then ```gulp example``` to run in the browser, ```jspm bundle src build/build.js``` or ```gulp bundle``` to build it.

It will fail because it can't find ```example-service.js``` but it should be looking for ```example-service.ts```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/systemjs/commit/ecc4a0734f00fe91e4d53c01e04e1816ce2521e9.

**Comment by frankwallis** [comment]
excellent, thank you

**Comment by guybedford** [comment]
Released in 0.19.1.
